### PR TITLE
SMTP Transporter Configuration for Mailer [Fixes #457 in Talawa Admin]  

### LINKED ISSUE #457: Fix Test `tests/user_organization.spec.js`
Our integrated test suites need to function correctly. This test file when run is failing and needs to be fixed.

- `tests/user_organization.spec.js`

Parent issue:
- https://github.com/PalisadoesFoundation/talawa-api/issues/449

### How to run the tests

To test you will need to start the test server, run the test, then stop the test server.

```
npm run start-test-server-in-action
npm run test
npm run stop-test-server-in-action
```

### PR DESCRIPTION
**Pull request by anshgoyalevil**
**What kind of change does this PR introduce?**
It adds the custom smtp server mailer support to talawa applications suite.

**Issue Number:**

Fixes PalisadoesFoundation/talawa-admin#457

**Did you add tests for your changes?**
Not Required

**Snapshots/Videos:**
image

**If relevant, did you update the documentation?**
TBD

**Does this PR introduce a breaking change?**
No

**Other information**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
1. Update INSTALLATION.md appropriately
2. Regenerate the Table of Contents

**Review by palisadoes — CHANGES_REQUESTED** [review]
1. Update INSTALLATION.md appropriately
2. Regenerate the Table of Contents

**Comment by anshgoyalevil** [comment]
@palisadoes 

I will do that within a few minutes.

I need to update the mailer.spec.ts file also.

File: src/utilities/mailer.ts [hunk 1]
```diff
@@ -13,23 +15,45 @@
...
+    transporter = nodemailer.createTransport({
+      host: String(SMTP_OPTIONS.SMTP_HOST),
+      port: Number(SMTP_OPTIONS.SMTP_PORT),
+      secure: SMTP_OPTIONS.SMTP_SSL_TLS === "true" ? true : false,
+      auth: {
+        user: SMTP_OPTIONS.SMTP_USERNAME,
+        pass: SMTP_OPTIONS.SMTP_PASSWORD,
+      },
+    } as SMTPTransport.Options);
+    console.log(SMTP_OPTIONS);
```
[the remark this answers is not in the document]
Comment (anshgoyalevil) [reply src/utilities/mailer.ts:32]: Resolved!

**Comment by anshgoyalevil** [comment]
@palisadoes 

Please review.

**Comment by anshgoyalevil** [comment]
Closed this due to GitGuardian Issue.

**Pull request closed without merging**